File: fib.py
import time

def fib(n):
    if n <= 1:
        return n
    else:
        return fib(n - 1) + fib(n - 2)

n = 35

start_time = time.time()
result = fib(n)
elapsed = time.time() - start_time

print(f"fib({n}) = {result}")
print(f"elapsed: {elapsed:.6f} secs")

File: fib.lua
function fib(n)
    if n == 0 then
        return 0
    elseif n == 1 then
        return 1
    else
        return fib(n - 1) + fib(n - 2)
    end
end

print("fib(35):")
local start_time = os.clock()
fib(35)
local elapsed = os.clock() - start_time
print("elapsed: " .. elapsed .. " secs")
